data/migrations/20201014204709_create_bare_bones_tables.js

```diff
@@ -12,7 +12,7 @@ exports.up = async (knex) => {
     table.varchar('zip_code').notNull();
     table.varchar('country').notNull();
     table.varchar('about', 5000);
-    table.varchar('hours');
+    table.varchar('hours', 5000);
```

Comment:
Sorry about that. Wasn't sure of the format you wanted in hours.

Reply:
No biggie I didn't know I needed more than the default 225 until I tried to insert the string.